app/controllers/api/v1/rents_controller.rb

```diff
@@ -4,23 +4,20 @@ class RentsController < ApiController
       before_action :set_locale
 
       def create
-        new_rent = current_user.rents.build
-        new_rent.assign_attributes(rents_params)
-
-        authorize new_rent
-
-        if new_rent.save
-          RentMailer.new_rent_notification(new_rent).deliver_later
+        rent = Rent.new(rents_params)
```

Comment:
You are instantiating a Rent twice, here and inside the interactor. I would remove this and move the Policy logic to UserPolicy, as we discussed the last week.